src/main/java/frc/robot/Robot.java

```diff
@@ -59,7 +62,18 @@ public void robotInit() {
 
         optionChooser = new SmartDashboardPathChooser();
         commandFactory = new CommandFactory(subsystemManager);
-        selfTestCommand = commandFactory.selfTestCommand();
+        //selfTestCommand = commandFactory.selfTestCommand();
+    
+        CameraServer inst = CameraServer.getInstance();
```

Comment:
this is enough code that i'd like to see it in a differnt method, pehaps called cameraInit().
also, a lot of these numbesr should be constants, probabl y in robot constants.

Did Cameraserver.getInstance().startAutomaticCapture() not work?